espresso-server/app/src/androidTest/java/io/appium/espressoserver/lib/model/SourceDocument.java

```diff
@@ -137,4 +137,38 @@ public static View findViewByXPath(String xpathSelector) throws XPathLookupExcep
             throw new XPathLookupException(xpathSelector, te.getMessage());
         }
     }
+
+    // Original Google code here broke UTF characters
+    private static String stripInvalidXMLChars(CharSequence charSequence) {
+        final StringBuilder sb = new StringBuilder(charSequence.length());
+        for (int i=0; i<charSequence.length(); i++) {
+            char c = charSequence.charAt(i);
+            if (XMLChar.isValid(c)) {
+                sb.append(c);
+            }
+        }
+
+        return sb.toString();
+    }
+
+    /**
+     * returns by excluding inner class name.
+     */
+    private static String simpleClassName(String name) {
```

Comment:
according to java spec methods names should contain verbs. so, this one can be changed to getSimpleClassName or extractSimpleClassName